src/control/server/storage/bdev/backend.go

```diff
@@ -434,13 +435,19 @@ func (sb *spdkBackend) formatAioFile(req *storage.BdevFormatRequest) (*storage.B
 		resp.DeviceResponses[path] = devResp
 		if err := createEmptyFile(sb.log, path, req.Properties.DeviceFileSize); err != nil {
 			devResp.Error = FaultFormatError(path, err)
+			os.Remove(path)
```

Comment:
Should not be needed to also remove the file when the chown and chmod are failing ?

Reply:
That's a good point, I think that it's okay to not remove the file if the permissions changes fail because they can be fixed by external admin intervention?

Reply:
The main reason for removing the file if the create fails as because otherwise a zero byte file with non-zero allocation is left.

Reply:
Not enough knowledge to define if it could be harmful to not remove such file with invalid permission => I will let you decide 😄

Reply:
@Michael-Hennecke do you have an opinion on this?

Reply:
@Michael-Hennecke votes on removing the file if any of the permission changes fail, I will make the change.